tests/epyccel/modules/generic_functions_2.py

```diff
@@ -137,6 +137,11 @@ def mix_types_3(x, y):
     return -x
 
 
+@template('T', types=['int64[:]', 'real[:]', 'complex[:]'])
+@types('T')
+def mix_array_scalar(x):
+    x[:] += 1
+
 @template('T', types=['int64[:]', 'real[:]', 'complex[:]'])
```

Comment:
I thought that `real` was deprecated in favor of `float`... :stuck_out_tongue_closed_eyes:

Reply:
Fixed for this file : 5b2dc7fa